Corona-Warn-App/src/test/java/de/rki/coronawarnapp/deadman/DeadmanNotificationOneTimeWorkerTest.kt

```diff
@@ -0,0 +1,65 @@
+package de.rki.coronawarnapp.deadman
+
+import android.content.Context
+import androidx.work.WorkerParameters
+import de.rki.coronawarnapp.worker.BackgroundConstants
+import io.kotest.matchers.shouldBe
+import io.mockk.MockKAnnotations
+import io.mockk.clearAllMocks
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.impl.annotations.RelaxedMockK
+import io.mockk.verify
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import testhelpers.BaseTest
+import testhelpers.coroutines.runBlockingTest2
+
+class DeadmanNotificationOneTimeWorkerTest : BaseTest() {
+
+    @MockK lateinit var sender: DeadmanNotificationSender
+    @MockK lateinit var context: Context
+    @RelaxedMockK lateinit var workerParams: WorkerParameters
+
+    @BeforeEach
+    fun setup() {
+        MockKAnnotations.init(this)
+    }
+
+    @AfterEach
+    fun teardown() {
+        clearAllMocks()
+    }
+
+    private fun createWorker() = DeadmanNotificationOneTimeWorker(
+        context = context,
+        workerParams = workerParams,
+        sender = sender
+    )
+
+    @Test
+    fun `create worker`() {
+        createWorker()
+    }
+
+    @Test
+    fun `run worker success`() = runBlockingTest2(permanentJobs = true) {
```

Comment:
If there was no specific issue that required this variant `runBlockingTest2(permanentJobs = true)`, use `runBlockingTest` or just add `suspend` to the test method.

`runBlockingTest2` is a workaround for coroutine scopes where a job never suspends (and never should).

Reply:
Edit to my previous comment, don't add `suspend` to test functions. It will not pick them up correctly.